frontend/src/index.css

```diff
@@ -1,9 +1,8 @@
+* {
+  font-family: 'Ubuntu';
```

Comment:
It's a better practice to set the `font-family` CSS property under the `body` selector since `body` is more specific than `*` so it has higher precedence. This could come in useful in the event we import another stylesheet that also sets the font-family property. 

If you look at the portion of code you deleted under the body selector, a fallback list of fonts (separated by commas) is provided in case the default font (in that case, -apple-system) is not found. Copy the existing list of fallback fonts and put it behind Ubuntu. The other two properties (-webkit-font-smoothing and -moz-osx-font-smoothing) are there just to make the fonts look better so be sure to add those back in too.

One more thing is that the Ubuntu font is never imported. You can add the [font stylesheet](https://fonts.google.com/specimen/Ubuntu?selection.family=Ubuntu) in `frontend/public/index.html`.